[RHCLOUD-35780] Make Drawer notifications actionable by adding a link to additional details (#4405)
RedHatInsights/notifications-backend: common-template/src/main/resources/templates/drawer/Vulnerability/newCveHighCvssBody.md

> Red Hat Insights has just identified a CVE with a CVSS score of >= 7. Please review the details of this [CVE]({environment.url}/insights/vulnerability/cves/{data.events[0].payload.reported_cve}?from=notifications&integration=drawer) and affected systems to determine risk and exposure for your organization.

Reviewer: ```suggestion
Red Hat Lightspeed identified a CVE with a CVSS score greater than 7. Review the details of [this CVE]({environment.url}/insights/vulnerability/cves/{data.events[0].payload.reported_cve}?from=notifications&integration=drawer) and affected systems to determine risk and exposure for your organization.
```
